src/celeritas/em/data/UrbanMscData.hh

```diff
@@ -35,7 +35,7 @@ struct UrbanMscParameters
     real_type range_fact{0.04}; //!< range_factor for e-/e+ (0.2 for muon/h)
     real_type safety_fact{0.6}; //!< safety factor
     real_type safety_tol{0.01}; //!< safety tolerance
-    real_type geom_limit{5e-8 * units::millimeter}; //!< minimum step
+    real_type geom_limit{5e-9 * units::millimeter}; //!< minimum step
```

Comment:
A good call!  I overlooked importance of the value of geom_limit - I thought that the value is chosen as somewhat arbitrarily small value near the safety limit, but should not have changed the value as it will affect other conditional criteria for MSC and we should consistently compare our implementation to that of Geant4 directly (with the exactly same value). Anyway, I recovered geom_limit back to the previous and temporarily introduced another arbitrary safety_limit for the boundary condition.